requirements.txt

```diff
@@ -19,6 +20,10 @@ Pillow==9.2.0
 protobuf==3.20.1
 pyparsing==3.0.9
 python-dateutil==2.8.2
+psycopg2==2.9.3
+python-socketio==5.7.1
```

Comment:
Remove it.

You can remove `venv` and reinstall old `requirements.txt` then just install the packages you needed and generate this file by `pip freeze > requirements.txt`.